src/Card.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components'
+import Box from './Box'
+import PropTypes from 'prop-types'
+
+function boxShadow(props) {
+
+  const boxShadows = {
+    'sm': {
+      'box-shadow': props.theme.boxShadows[0]
+    },
+    'md': {
+      'box-shadow': props.theme.boxShadows.slice(0,2).join(', ')
+    },
+    'lg': {
+      'box-shadow': props.theme.boxShadows.slice(0,3).join(', ')
+    },
+    'xl': {
+      'box-shadow': props.theme.boxShadows.join(', ')
+    }
+  }
+  return boxShadows[props.boxShadowSize]
+}
+
+const Card = styled(Box)`
+  ${boxShadow}
```

Comment:
It might make sense to add `${borderColor}` possibly with a default color of `borderGray`.  I know it makes sense for the upcoming summary of charges from the packages team.

Reply:
Or maybe `borderColor` belongs on `Box`?

Reply:
@jes708: Probably make sense to add it to `Box` esp. if we have a default shadow for `Card`. I can add it to `Box` if everyone agrees.

Reply:
That sounds right to me.

Reply:
My thought is to let `card` give the border or shadow. Let `box` handle the layout and `card` handle styling.

Reply:
@odmln I agree, I would say borderColor belongs on Card not on Box. But a default gray borderColor might be nice.

Reply:
I like that reasoning

Reply:
Cool, I'll add `border` to Card. Should there be a default border size?  @odmln: What should the default color for border be?

Reply:
@cenjennifer I've been using `box-shadow: 0 0 0 1px rgba(0,0,0,0.08)` instead of a pure border for things like cards, popovers, and tooltips. But if thats too light we do now have `borderGray #abb7c2` in the design system that you could use. I don't think cards will really be using different color borders, it's only inputs that will change.

Reply:
I made border a defaultProp, but not 100% sure if border and box-shadow are separate styles that can be displayed individually in a card component @odmln